src/dns-test.js

```diff
@@ -96,6 +110,20 @@ var dnsAttempts = {
     tcpNEWFOUR:  0
 };
 
+/**
+ * Shuffle the order of DNS queries we issue
+ */
+function shuffleQueries(queries) {
```

Comment:
This function logic looks equivalent to https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array/2450976#2450976. It might be a coincidence, but if it was indeed copied from there, let's add a reference to it.

While this function is indeed being used to shuffle queries, the logic is independent of the thing to shuffle. So using the original name (`array` instead of `queries`) would make sense. It does make sense to be somewhat more clearer than the given function name (use `shuffleArray` instead of the generic `shuffle`).